ingest/sqs/ingester.go

```diff
@@ -37,11 +37,6 @@ func (i *sqsIngester) ProcessBlock(ctx sdk.Context) error {
 
 	goCtx := sdk.WrapSDKContext(ctx)
 
-	// Begin by flushing all previous writes
-	if err := tx.ClearAll(goCtx); err != nil {
-		return err
-	}
-
```

Comment:
Note; we don't need to clear all data every block. It can be simply overwritten with the current design.

This will allow caching routes across blocks to minimize recompute